src/inventory_client/inventory_client.go

```diff
@@ -156,10 +154,13 @@ func (c *inventoryClient) getEnabledHostsWithInventoryInfo() (map[string]Enabled
 }
 
 // TODO move states to enums after bm-inventory changes
-func (c *inventoryClient) SetConfiguringStatusForHosts(inventoryHostsMapWithIp map[string]EnabledHostData, mcsLogs string) {
-	notValidStates := map[string]string{configuring: "", "Joined": "", "Done": ""}
+func (c *inventoryClient) SetConfiguringStatusForHosts(inventoryHostsMapWithIp map[string]EnabledHostData, mcsLogs string, fromBootstrap bool) {
+	notValidStates := map[models.HostStage]string{models.HostStageConfiguring: "", models.HostStageJoined: "", models.HostStageDone: ""}
```

Comment:
since you want to use notValidState as set, you can define it: map[models.HostStage]{}
this is a clearer, cleaner way, and does not wste memory ( not very important in this case :) )

Reply:
sure, didn't know about this option